app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command):
+    ls = command.split()
+    if ls[1] != ls[2]:
+        with open(f"{ls[1]}", "r") as f1:
+            content = f1.read()
```

Comment:
Best practice read line by line, because if you have a large file, your computer and memory won’t be happy.